apps/docs/content/components/Avatar.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Avatar
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=543%3A19830&t=Be91VGjX2BANJnQc-11'
+source: https://github.com/primer/brand/tree/main/packages/react/src/Avatar/Avatar.tsx
+storybook: '/brand/storybook/?path=/story/components-Avatar--playground'
+description: Use Avatar to display a thumbnail representation of a person.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Avatar, Stack} from '@primer/react-brand'
```

Comment:
```suggestion
import {Avatar} from '@primer/react-brand'
```

Reply:
Stack is used for [some of the previews](https://github.com/primer/brand/blob/a348157861960b4ed7c0296957b4ebc1d9c8373c/apps/docs/content/components/Avatar.mdx?plain=1#L29)